ocpp/v20/call.py

```diff
@@ -0,0 +1,421 @@
+from dataclasses import dataclass
+from typing import Any, Dict, List
+
+
+@dataclass
+class AuthorizePayload:
+    id_token: Dict
+    _15118_certificate_hash_data: List = None
+    evse_id: List = None
+
+
+@dataclass
+class BootNotificationPayload:
+    charging_station: Dict
+    reason: str
+
+
+@dataclass
+class CancelReservationPayload:
+    reservation_id: int
+
+
+@dataclass
+class CertificateSignedPayload:
+    cert: List
+    type_of_certificate: str = None
+
+
+@dataclass
+class ChangeAvailabilityPayload:
+    evse_id: int
+    operational_status: str
+
+
+@dataclass
+class ClearChargingProfilePayload:
+    evse_id: int = None
+    charging_profile: Dict = None
+
+
+@dataclass
+class ClearDisplayMessagePayload:
+    id: int
+
+
+@dataclass
+class ClearVariableMonitoringPayload:
+    id: List
+
+
+@dataclass
+class ClearedChargingLimitPayload:
+    charging_limit_source: str
+    evse_id: int = None
+
+
+@dataclass
+class CostUpdatedPayload:
+    total_cost: int
+    transaction_id: str
+
+
+@dataclass
+class CustomerInformationPayload:
+    request_id: int
+    report: bool
+    clear: bool
+    customer_certificate: Dict = None
+    id_token: Dict = None
+    customer_identifier: str = None
+
+
+@dataclass
+class DataTransferPayload:
+    vendor_id: str
+    message_id: str = None
+    data: Any = None
+
+
+@dataclass
+class DeleteCertificatePayload:
+    certificate_hash_data: Dict
+
+
+@dataclass
+class FirmwareStatusNotificationPayload:
+    status: str
+    request_id: int
+
+
+@dataclass
+class Get15118EVCertificatePayload:
+    _15118_schema_version: str
```

Comment:
This one I wanted to discuss with you. I added an underscore before the number, however I dont know if when we convert from camel case to snake case if it will convert it with the underscore or not

Reply:
We discussed offline that we don't like it, but also have not better solution right now.